### ISSUE TITLE
Cannot login with correct handle and password

### ISSUE THREAD

**Issue by kali4128722545** [bluesky-social/social-app#4961]
**Describe the bug**
Hi, when i try to log into https://bsky.app/ it says "Invalid identifier or password". I haven't changed anything, it worked just yesterday. I tried to reset the password like 10 times now with different passwords, none of it works.

**To Reproduce**

1. Try to log in?

**Expected behavior**

To be actually logged in, which i am not.

**Screenshots**

![Screenshot_1](https://github.com/user-attachments/assets/2c741d25-7b92-45e8-9462-3cb3eb2ba46b)

**Details**

 - Platform: Desktop, i tried 3 different OS, Windows 7, Linux, Android. None works.
 - Platform version:
 - App version:

**Additional context**

**Comment by surfdude29** [comment]
Hi @kali4128722545 sorry to hear you're having problems logging in. You can email support here for help:

https://blueskyweb.zendesk.com/hc/en-us/requests/new

**Comment by kali4128722545** [comment]
Well, they really don't? They just asked me if i have typed my password and email correctly, which is sort of taking the user for stupid? I use a password manager. I changed the password 20 times now and still can't log in. And part of your website gives me 404 and 401 to begin with. You can believe me that this is not a me-problem...
![Screenshot_5](https://github.com/user-attachments/assets/b8b97937-3fb7-4d4d-a82a-8b07b199e330)
![Screenshot_7](https://github.com/user-attachments/assets/ad3c1357-0007-47a4-a95a-cd5bd312d203)

**Comment by surfdude29** [comment]
To clarify, I don't work for Bluesky or anything, I'm just a fellow user and I was trying to help by passing along what @/pfrazee said when I asked what the best way is to get support for a login issue:

https://bsky.app/profile/pfrazee.com/post/3l26gnjjlpt2w

The only thing I can think of suggesting is creating a new Bluesky account and asking one of the devs via that account (and linking to this Github issue).

**Comment by kali4128722545** [comment]
Ok, thank you anyway, man. I just took it here bc so many people seem to have the exact same problem.

https://www.google.com/search?q=bluesky+can%27t+sign+in+site%3Awww.reddit.com&sca_esv=8c3c0483a16c4f3c&biw=1440&bih=799&ei=KyDFZrIn-pX27w_X0_W4Dg&ved=0ahUKEwjy6OCw1oSIAxX6iv0HHddpHecQ4dUDCA8&uact=5&oq=bluesky+can%27t+sign+in+site%3Awww.reddit.com&gs_lp=Egxnd3Mtd2l6LXNlcnAiKWJsdWVza3kgY2FuJ3Qgc2lnbiBpbiBzaXRlOnd3dy5yZWRkaXQuY29tSABQAFgAcAB4AJABAJgBAKABAKoBALgBA8gBAJgCAKACAJgDAJIHAKAHAA&sclient=gws-wiz-serp

**Comment by surfdude29** [comment]
Yeah sign-in issues are especially frustrating because you can't even log into the platform to ask for help :/

If you'd prefer not to create a temporary second account (which I completely understand) and ask the devs, one other thing that might be worth trying: have all your login/password reset attempts been over the same connection (e.g. WiFi perhaps)? If you haven't tried this already, maybe it's worth trying over a mobile data connection or something, just in case that gets around the issue?

**Comment by kali4128722545** [comment]
I don't see how that fixes an HTTP error, I am sorry.

Am Mi., 21. Aug. 2024 um 01:11 Uhr schrieb surfdude29 <
***@***.***>:

**Comment by mike-silva** [comment]
I have the exact same problem. At first I thought it was because of some bug that precludes logging in from two instances on the same device, as it is highly reproducible that I am logged into the app on one device, but can't login from the browser on the same.

However, that's not the issue. As I can be logged into the app on my phone, but unable to login to it via the app or browser on my Mac.

If I do a password reset, I can then login to one instance on one device with it, before the same shitty bug rears its head.

**Comment by mike-silva** [comment]
A link to show that this is causing problems for many people and has been for some time.
https://www.reddit.com/r/BlueskySocial/comments/15faqty/invalid_identifier_or_password_on_android_mobile/

**Comment by d3vv3** [comment]
I am also facing this issue. Using a password manager as well.

**Comment by PeanutButterJellyTimez** [comment]
I'm having the same issue. I am logged into an instance on my phone, and when I go to use it on my laptop: no dice ("Invalid identifier or password"). Must create new password (I use a password manager: forgetting the pswd isn't the problem), with which I am logged into my laptop now, and then phone instance no longer works (rinse, repeat). Can use it for awhile until I want to use it on the other device. Very annoying.

**Comment by niemasd** [comment]
I'm running into the same issue. I'm using Bitwarden on my laptop (Windows 11, Chrome) and phone (Android) to randomly generate password:

1. Used "Forgot Password" on my laptop to reset the password, was able to sign in with the new password on my laptop, but then wasn't able to sign in with the same password on the Android app
2. Then used "Forgot Password" on the Android app to reset the password again, was able to sign in with the new password in the Android app, but then wasn't able to sign in with the same password on my laptop
3. Then used "Forgot Password" on my laptop to reset the password again, was able to sign in with the new password on my laptop, but then wasn't able to sign in with the same password on the Android app

Interestingly, I have a script that uses the Python API to do stuff, and I *am* able to sign in via the Python API with this exact username and password, so I know it's correct (and I'm copy-pasting, so there's no typos; I also tried manually typing it into the Android app just in case there's an issue with copy-paste and got the same result)

**Comment by PeanutButterJellyTimez** [comment]
Interesting! I am using the same combo (Bitwarden, Android phone, Chrome).

Ok, I think I might've solved the mystery; though, I don't understand the secret sauce at-play. Last night I had to reset my pswd again to check my Bsky on my phone after having used it on my computer all day. I used the Bitwarden generator, saved the new one, and we're off to the races. 

This morning I go to input it in my laptop using the bitwarden instance that shows up, and no dice. However, using the _search_ function in Bitwarden, I identify **two** instances of Bsky: _bsky.app_ and _xyz.bsky.app_. Only bsky.app shows up when I go to auto-input on my laptop, and only xyz.bsky.app when I go to input on my phone. Curiously, **both have the same password**. When I use the xyz.bsky.app instance in both my phone AND on my laptop, I can get in without resetting. It's a mystery to me why that works. Let me know if that works for you.

**Comment by niemasd** [comment]
Yup, same for me! When I checked my phone's Bluesky Android app after seeing your reply, I was signed out in the app, and when I tried to sign in using Bitwarden on my phone (even just clicking autofill), it worked perfectly now. So it was probably some server-side issue that they've somehow fixed now. Glad yours is working as well 😄

**Comment by kali4128722545** [comment]
Yeah i will report them to the european data protection agencies. This
violates gdpr article 7.

Niema Moshiri ***@***.***> schrieb am Fr., 15. Nov. 2024,
22:07:

**Comment by kali4128722545** [comment]
Yxz.bluesky.app is not a registered dpmain for me

Nina Wilke ***@***.***> schrieb am So., 17. Nov. 2024, 09:46:

**Comment by PeanutButterJellyTimez** [comment]
Two instances in bitwarden for me are now: 
Bsky.app
xyz blueskyweb.app

As I'm not familiar with this stuff, why is any of this a violation of gdpr article 7? Thanks in advance!

**Comment by kali4128722545** [comment]
Wow. How quickly things move when you threaten GDPR. You are no better than
X. Have a nice day and good luck with this UX nightmare of yours.

Nina Wilke ***@***.***> schrieb am So., 17. Nov. 2024, 09:46:

**Comment by kali4128722545** [comment]
ARE YOU KIDDING! You don't know the laws you have to be compliant with? I
have to be able to delete my account. You could have googled that! WHAT A
JOKE

PeanutButterJellyTimez ***@***.***> schrieb am So., 17. Nov.
2024, 10:52:

**Comment by PeanutButterJellyTimez** [comment]
I did Google it but still didn't understand its application to this thread (I'm not in whatever field you assume I'm in, nor am I a bsky rep, I'm just a human who had log-in issues and found a thread where others were having the same issue), which is why I politely asked. 

None of us was talking about deleting our Bluesky.social accounts, we were discussing a log-in quirk. Please double-check the thread, possibly you meant to comment on something else?

**Comment by kali4128722545** [comment]
well, if i want to delete it  first of all i have to able to log in,
right. Apart from all of that, i just made another account and i am having
the same problem when i get asked my password from account deletion. Hence
now you have personal data from 2 on my accounts which i cannot delete.

Am So., 17. Nov. 2024 um 14:30 Uhr schrieb PeanutButterJellyTimez <
***@***.***>:

**Comment by mike-silva** [comment]
I think one of two things is going on here:

- The app and website have different domains. We are creating passwords saved against one domain, then trying to fill an old password from the other. i.e. See if you have two Bluesky logins saved in your password manager. Consolidate if so.

- There is something wonky going on with the App password implementation. In my case, after I created an app password for my browser on my laptop, via the app on my phone, I could login from my laptop browser. Good, but not good enough though, as you don't have full permissions in the laptop browser. I then went about signing out from the app password on my laptop browser, and signing in from the main account password and voila, it now worked.

My stupid conjecture being, perhaps for those accounts created before app passwords were implemented, there's a bug where you have to go create and login from one, before you can use your main account password on more than one instance [whether browser or app].

**Comment by patientx** [comment]
Same problem here.

**Comment by markbrewster** [comment]
Same problem here

**Comment by zhukoz** [comment]
I have never been able to log in via PC due to this issue. Not via any browser either, not just the windows app; so I am only able to access bluesky via mobile - and I don't dare change the password in case I lose access altogether. Email or handle login makes no difference

**Comment by markbrewster** [comment]
It seemed to be ok if I typed in a password. Cutting/pasting or using password manager caused problems.

**Comment by zhukoz** [comment]
> It seemed to be ok if I typed in a password. Cutting/pasting or using password manager caused problems.

Glad if that solution worked for you. Alas not for me unfortunately.

**Comment by AndreiMiculita** [comment]
Also encountering the same issue. Have tried resetting my password twice. One of them by manually typing it in. No luck. Looks like I'm locked out of it.

A bit nuts that there is neither a password confirmation field nor a button to unhide a password while typing in. Didn't get an account creation confirmation email either (but I did get some emails when resetting the password).

I guess I'll be giving Bluesky a pass.

**Comment by zhukoz** [comment]
> Had the same problem. And finally made it work! In Bluesky mobile application go to Settings -> Privacy&Security and choose App passwords. Then create separate password for your web browser access and login with it on bsky.app page. It worked! EDIT: I (too) used strong generated password from my Android Chrome browser. It might be an issue here as the application specific passwords are in form xxxxx-xxxxx-xxxxx-xxxxx (and no special chars)

I'm glad it worked for you! Alas it didn't for me, at all lol

I just ended up using a completely new email and creating a second account, which of course isn't ideal as all my followers and posts are completely separate, but what else can you do? Bluesky support were completely useless lol

**Comment by zhukoz** [comment]
> Had the same problem. And finally made it work! In Bluesky mobile application go to Settings -> Privacy&Security and choose App passwords. Then create separate password for your web browser access and login with it on bsky.app page. It worked! EDIT: I (too) used strong generated password from my Android Chrome browser. It might be an issue here as the application specific passwords are in form xxxxx-xxxxx-xxxxx-xxxxx (and no special chars)

I'm glad it worked for you! Alas it didn't for me, at all lol

I just ended up using a completely new email and creating a second account, which of course isn't ideal as all my followers and posts are completely separate, but what else can you do? Bluesky support were completely useless lol

**Comment by KenKaneki18** [comment]
I have the same issue but I don't use any password client for Bluesky. I didn't change my password and was logged out on my Chromebook

This is the second time though that I have to change my password in order to be able to login again

It's really weird that you're actually blocked from logging in unless you change your password again

**Comment by vgiotsas** [comment]
I have exactly the same issue

**Comment by worldsayshi** [comment]
Check for duplicate entries in your password manager. It's easy to accidentally end up with one entry per device. Delete all entries, create a new entry with the correct password. Synk all devices. Make sure to use the same entry.

**Comment by doc-bok** [comment]
I get the same problem. Every time I log in on a different device I _have_ to do a password reset. Every time I do, I literally update the password to the one saved in my password manager and am able to log in. I've been using the exact same password for several password resets, and yet I can **never** log in unless I go through the password reset process first.

**Comment by estrattonbailey** [comment]
Hey y'all, sorry for the radio silence on this one.

If you're still having this issue, can you 👍 this comment? If you feel comfortable with it, feel free to share your handle, as it will help us look up your account. If you'd rather not, please [submit a support request](https://blueskyweb.zendesk.com/hc/en-us/requests/new) using the email on your account and mention this Issue.

Also, please make sure that you're typing or copy/pasting in your password (in case your password manager is doing something funky), and that you're using your full handle e.g. `handle.com` or `handle.bsky.social`, etc.

**Comment by psychologybromail-spec** [comment]
It’s 2026, still encountering this problem. Ridiculous.

**Comment by robinatcoolth** [comment]
I'm having this problem. I want to use the app on my phone and the web interface on my laptop. I use Bitwarden. When I try to log in on the second device I am told the password is wrong. I use the forgot password link and then enter my existing password as the new password. I can then log in on the new device. When I then go to the other device It doesn't like the password again so I go through the same forgot password route and can then log in. I am continuing to use the same password but can only access via one route at a time.

**Comment by tobble** [comment]
I'm getting the same "Incorrect username or password"  trying to login through Firefox browser on both Windows laptop and Android phone but can on Edge and the app. I've deleted cookies and site data but that doesn't seem to help. Reluctant to delete all site preferences unless someone says it might help. Tried removing tracking protection. No dice.

I've just logged out and in again on Edge to check it's not timed out after multiple password failures. No Bitwarden here. Using password manager so so possibility of typing error.

**Comment by schumaml** [comment]
@estrattonbailey I have also encountered this, and finally found that this is a known problem. Is there any chance of this getting fixed once and for all, or does this require some manual work on a case-by-case basis, as your previous comment implies?
